### DISCUSSION TITLE
Does chia syncing slow down plotting?

### DISCUSSION THREAD

**Discussion by proud2blds** [swar/Swar-Chia-Plot-Manager#651] [Q&A]
Hi all, does anyone know of node syncing affects plotting times. I have a Dell T5810 that hasn't finished syncing yet with xeon e52699v3 64 ddr4 4tb inland platinum nvme. Plotting in parallel 9 plots, 5 threads, 60 min stagger, max 4 in phase 1 and it takes 13 to 15 hours per plot. I have another i710700k similar specs fully synced and only takes about 8 hours.

**Comment by Rayrayray87** [comment]
NO node sync wont slow down plotting, although your completed plots will not farm until you are fully synced.
You should read this guide to plotting on multiple machines and follow it, after read through Chia's official github- really great info.
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines 

Your inland platinum is the bottle neck it's cache is crap, it slows down to sata speed when you throw more than 4 plots at it, 
or 2 in phase 1.
You MUST get the Inland Premium and you are much better off with 2- 2TB drives than 1- 4TB drive. Uses more PCI lanes but that's okay that proc has 40. Stripe the 2 drives so you get better performance.  You could also get 2- 1TB Hynix Gold P31 Nvme- they are lighting fast with high endurance, I use 6 of them in my server. 

Also remember most of plotting is single thread; phase 2-5 are all single threaded. Your i7-10700K has better Single core performance than the Xeon 2.3ghz. Turbo boost wont continuously boost all cores FYI.  
**OR IF ON A BUDGET***
Modify your config to only plot 2 in Phase 1 with 6 Threads 
Increase the Stagger time to 120Mins (drop to 90Min if you notice plots are finishing around 8hrs)
Set ram budget to 3432
Max concurrent 8
That should get you down to 9-10 hrs per plot. 
Hope it helps.
